examples/connext_dds/builtin_topics/py/README.md

```diff
@@ -0,0 +1,25 @@
+# Example Code: Group Coherent Presentation
+
+If you haven't used the RTI Connext Python API before, first check the
+[Getting Started Guide](https://community.rti.com/static/documentation/connext-dds/current/doc/manuals/connext_dds_professional/getting_started_guide/index.html).
+
```

Comment:
Can we add an explanation of what the example does. Explaining that:
* We access the `ParticipantBuiltinTopicData` samples sent by a remote participant
* We are storing a "password" in the user data field of these builtin topic samples
* If the passwords don't match, we ignore the remote participant - preventing communication

Reply:
That should be explained in the top-level README. That's how the examples are usually structured. The README in the top-level folder explains the example concept and each language's folder deals with the specific instructions to build and run it for that language.